### PR TITLE
Added list of supported scrapes to top of page to save reading the wh…

### PR DESCRIPTION
**Pull request by Alex3k**
**What this PR does / why we need it**:
I have added list of supported scrapes to top of page to save reading the whole thing. This provides a simple way for me to provide a link to a list of supported inputs rather than ask customers to read the whole page. Much easier for sharing information

### CODE REVIEW

File: docs/sources/clients/promtail/scraping.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ description: Promtail Scraping (Service Discovery)
 ---
 # Scraping
 
+Promtail currently supports scraping from the following sources:
+- Files
```
Comment (JStickler) [inline docs/sources/clients/promtail/scraping.md:8]: Why don't we go ahead and make each bullet point a hyperlinks and save some more scrolling?
```suggestion
- {{< relref "#file-target-discovery" >}}
```

File: docs/sources/clients/promtail/scraping.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ description: Promtail Scraping (Service Discovery)
...
+- Kubernetes Discovery
```
Comment (JStickler) [inline docs/sources/clients/promtail/scraping.md:9]:
```suggestion
- {{< relref "#kubernetes-discovery" >}}
```

File: docs/sources/clients/promtail/scraping.md [hunk 3]
```diff
@@ -4,6 +4,17 @@ description: Promtail Scraping (Service Discovery)
...
+- Journal (Linux Only)
```
Comment (JStickler) [inline docs/sources/clients/promtail/scraping.md:10]:
```suggestion
-  {{< relref "#journal-scraping-linux-only" >}} 
```

File: docs/sources/clients/promtail/scraping.md (hunk 2 above)
Comment (JStickler) [reply docs/sources/clients/promtail/scraping.md:9]: Hmmm, I'm looking at the right hand navigation/table of contents.  Is `Kubernetes Discovery` at the correct heading level?  Should it be under `File Target Discovery` or at the same level?

**Comment by MasslessParticle** [comment]
Closing for inactivity

**Comment by Alex3k** [comment]
Why was this closed for inactivitiy @MasslessParticle ? I was waiting for someone to merge it

**Comment by Alex3k** [comment]
Why was this closed for inactivitiy @MasslessParticle ? I was waiting for someone to merge it

**Comment by JStickler** [comment]
@Alex3k  Before it can be merged, this PR needs to be updated and the generated documentation needs to be updated by running 'make doc' and committing the changes.

**Comment by JStickler** [comment]
@Alex3k Could you resolve the conflicts on this PR so we can get it merged?

**Comment by JStickler** [comment]
Closing due to inactivity and merge conflicts, replaced by #11444 .

**Pull request closed without merging**